cmake/Modules/FindIrrlicht.cmake

```diff
@@ -29,8 +29,22 @@ foreach(libname IN ITEMS IrrlichtMt Irrlicht)
 	endif()
 endforeach()
 
-# Users will likely need to edit these
-mark_as_advanced(CLEAR IRRLICHT_LIBRARY IRRLICHT_INCLUDE_DIR)
+# Handholding for users
+if(IRRLICHT_INCLUDE_DIR AND (NOT IS_DIRECTORY "${IRRLICHT_INCLUDE_DIR}" OR
+	NOT EXISTS "${IRRLICHT_INCLUDE_DIR}/irrlicht.h"))
```

Comment:
Missing tab, also on L42

Reply:
Do we also apply that code style to cmake?

Reply:
Good question actually :shrug: 

well, it's nitpicking, so just keep it as-is.